.gitignore

```diff
@@ -12,8 +12,10 @@ build/
 .idea/
 build-eclipse/
 /tags
+out/
```

Comment:
Any reason for this?

Reply:
IntelliJ puts its output under the out/ directory, rather than the build/ directory, despite supposedly auto-updating from the gradle files.